src/runtime/Function.cpp

```diff
@@ -149,4 +149,49 @@ void ImportedFunction::call(ExecutionState& state, Value* argv, Value* result)
     m_callback(newState, argv, result, m_data);
 }
 
+WasiFunction* WasiFunction::createWasiFunction(Store* store,
+                                               FunctionType* functionType,
+                                               WasiFunctionCallback callback,
+                                               Instance* instance)
```

Comment:
It seems that parameter `instance` is always nullptr when `WasiFunction` is first created.
It would be better to remove invalid `instance` parameter here.

Reply:
I have removed the invalid `instance` and instead initialize the variable to nullptr when a `WasiFunction` is created.